generate_report.py

```diff
@@ -0,0 +1,46 @@
+from drift import info_parser
+from drift.app import create_app
+import json
+
+from kerlescan.inventory_service_interface import interleave_systems_and_profiles
+
+print("Testing report generation...")
+# TODO: remove need to initialize flask app
+app = create_app()
+app.app.app_context().push()
+
+systems, profiles, baselines, hsps = [], [], [], []
+with open("tests/reports/inputs/inventory/report1-systems.json", "r") as myfile:
+    systems = json.loads(myfile.read())["results"]
+
+with open("tests/reports/inputs/inventory/report1-system_profiles.json", "r") as myfile:
+    profiles = json.loads(myfile.read())["results"]
+
+systems_with_profiles = interleave_systems_and_profiles(systems, profiles)
+
+with open("tests/reports/inputs/inventory/report1-system_profiles.json", "r") as myfile:
+    profiles = json.loads(myfile.read())["results"]
+
+with open("tests/reports/inputs/baselines/report1-baselines.json", "r") as myfile:
+    baselines = json.loads(myfile.read())["data"]
+
+with open("tests/reports/inputs/hsps/report1-hsps.json", "r") as myfile:
+    hsps = json.loads(myfile.read())["data"]
+
+comparisons = info_parser.build_comparisons(
+    systems_with_profiles, baselines, hsps, None
+)
+
+
+# to test generated report against expected data:
+with open("tests/reports/outputs/report1.json", "r") as myfile:
+    result = json.dumps(comparisons, indent=4, sort_keys=True).splitlines()
```

Comment:
You used sort_keys=True here which will alphabetize them but you do not alphabetize the 'expected' json below so this comparison will probably fail.

Reply:
it is alphabetized when saving from the `sort_keys=True` in the section below that writes the report. I guess I can add a command-line option to write a new report JSON instead of doing it the current way of commenting/uncommenting, i'll do that tomorrow.